src/components/ContentHighlights/HighlightStepper/ContentHighlightStepper.jsx

```diff
@@ -78,7 +81,7 @@ const ContentHighlightStepper = ({ enterpriseId }) => {
         isPublished: result.isPublished,
         title: result.title,
         uuid: result.uuid,
-        highlightedContentUuids: [],
+        highlightedContentUuids: result?.highlightedContentUuids || [],
```

Comment:
nit: `result` is always defined in the other object properties. why use `result?.` here?